std/haxe/xml/Fast.hx

```diff
@@ -106,17 +106,106 @@ private class NodeListAccess implements Dynamic<List<Fast>> {
 
 }
 
+/**
+	The `haxe.xml.Fast` API helps providing a fast dot-syntax access to the 
+	most common `Xml` methods.
+**/
 class Fast {
-
-	public var x(default,null) : Xml;
+	/**
+		The current corresponding `Xml` node.
+	**/
+	public var x(default, null) : Xml;
+
+	/**
+		The name of the current element. This is the same as `Xml.nodeName`.
+	**/
 	public var name(get,null) : String;
+
+	/**
+		The inner PCDATA or CDATA of the node. 
+		
+		An exception is thrown if there is no data or if there not only data
+		but also other nodes.
+	**/
 	public var innerData(get,null) : String;
+
+	/**
+		The XML string built with all the sub nodes, excluding the current one.
+	**/
 	public var innerHTML(get,null) : String;
+
+	/**
+		Access to the first sub element with the given name. 
+		
+		An exception is thrown if the element doesn't exists.
+		Use `hasNode` to check the existence of a node.
+		
+		```
```

Comment:
I mean after the triple backticks. It seems that one of the code blocks you added has `haxe` there, and the others don't.